src/components/staffing/ProjectCell.js

```diff
@@ -2,23 +2,41 @@ import React from 'react'
 import { Cell } from 'fixed-data-table'
 
 export default class ProjectCell extends React.Component {
-
   static propTypes = {
     data: React.PropTypes.array.isRequired,
     field: React.PropTypes.string,
     rowIndex: React.PropTypes.number,
     onClick: React.PropTypes.func,
-  }
+  };
+  ignoredProjectNames = ['HOLIDAY', 'NO PROJECT NAME', 'Leave UK', 'NOTHERE'];
 
   render() {
     const { rowIndex, field, data, ...props } = this.props
+    const currentWeek = Object.keys(data[rowIndex].staffing).sort()[0]
```

Comment:
It's dangerous to do that in one line because the "sort" function does not follow "functional programming principles". It modifies the "input array" whereas when doing FP, you expect the inputs to be untouched and the returned value to be the correct one.

In this case it still works because it turns out that "sort" also returns the reference to the input array (see [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort))
→ But it means the readability is deteriorated (a dev reading this will wonder what happens and waste a little bit of time)

Any idea how to make this code easier to read?